CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 1.11.0 - 2023-07-15
```

Comment:
The CHANGELOG needs to be under the unreleased and later all this is automatically updated and moved onto the version. This is mistake on my end since I haven't specified any guidelines for this. Apologies